saltpylint/__about__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+
+__title__ = 'SaltPyLint'
+__description__ = 'Required PyLint plugins needed in the several '\
+    'SaltStack projects.'
+
+__version_info__ = (2017, 5, 19)
+__version__ = '{0}.{1}.{2}'.format(*__version_info__)
```

Comment:
So the real thing that doesn't make sense to me is having the package metadata in like 3 places (constants in `setup.py`, dunders in `version.py`, and other stuff in the `setup` call).

Absent any codified standard I borrowed `__about__.py` from several projects authored/owned by the Python Packaging Authority. See here:

https://github.com/pypa/packaging/blob/master/packaging/__about__.py
https://github.com/pypa/warehouse/blob/master/warehouse/__about__.py
https://github.com/pypa/history/blob/master/history/__about__.py
https://github.com/pypa/readme_renderer/blob/master/readme_renderer/__about__.py

 _Personally_ I would just store everything in `setup.py`. If there is a compelling reason to have the version in the module code itself than putting it in `__version__` in `__init__.py` and leaving everything in the `setup()` call seems pretty sane. The more I think about it the more inclined I am to think it makes sense to simply move all but the `classifiers` and `packages` into `__about__.py`.